index.bs

```diff
@@ -2632,10 +2632,11 @@ run the following steps:</p>
 
       <ol>
 
-       <li><p>If |value| contains a U+002C COMMA character (,), then let |linepos| be the leading
-       substring of |value| up to and excluding the first U+002C COMMA character (,) in that string
-       and let |linealign| be the trailing substring of |value| starting from the character
-       immediately after the first U+002C COMMA character (,) in that string.</p></li>
+       <li><p>If |value| contains a U+002C COMMA character (,) that is not the last character in
```

Comment:
If we want the comma version to take priority over the non comma version, we should allow this as one where the parameters take the default value, I.e. linealign=start, seeing as if we make this extensible with further parameters, we will not always expect all the parameters to be present.
E.g. line:50,XYZ=blah should still take precedent over line:50 and result in linealign getting the default value.